SmoothScroll.js

```diff
@@ -415,13 +415,13 @@ function keydown(event) {
     // [spacebar] should trigger button press, leave it alone
     if ((isNodeName(target, 'button') ||
          isNodeName(target, 'input') && buttonTypes.test(target.type)) &&
-        event.keyCode === key.spacebar) {
+        (event.key === '(Space character)' || event.key === 'Spacebar' || event.key === ' ') {
```

Comment:
```diff
- (event.key === '(Space character)' || event.key === 'Spacebar' || event.key === ' ') {
+ (event.key === '(Space character)' || event.key === 'Spacebar' || event.key === ' ')) {
```

Reply:
The reason for three methods for spacebar is because of the following:

- Future propsoal - `(Space character)`
- Current proposal - ` `
- Passed propsal (older browsers e.g. IE11) - `Spacebar`

Reply:
Thanks for explaining :smiley: but I was saying you missed a bracket `)`

Reply:
Thanks 👍